core/class/CommandLineTool.go

```diff
@@ -185,6 +190,41 @@ func (tool *CommandLineTool) placeInputFilesToCommandExecDir() error {
 	return nil
 }
 
+// defineStdoutDestination
+func (tool *CommandLineTool) defineStdinSource() error {
+	if tool.Root.Stdin != "" {
+		stdin, err := tool.Command.StdinPipe()
+		if err != nil {
+			return err
+		}
+		defer stdin.Close()
+		stdinfilename := tool.Root.Stdin
+		if strings.HasPrefix(stdinfilename, "$(") && strings.HasSuffix(stdinfilename, ")") {
+			// Create JavaScript runtime
+			vm, err := tool.Root.Inputs.ToJavaScriptVM(tool.Command.Dir)
```

Comment:
no need to pass `tool.Command.Dir` to `ToJavaScriptVM`

Reply:
Thanks !